src/createInstancePanel.ts

```diff
@@ -0,0 +1,143 @@
+    nodeCount: number = 1;
+}
+
+export class CreateInstancePanel {
+
+    private readonly panel: vscode.WebviewPanel;
+    private readonly rpcServerExplorer: RpcServerExplorer;
+
+    private viewState: ViewState;
+
+    private disposed: boolean = false;
+    private created: boolean = false;
+
+    constructor(
+        extensionPath: string,
+        workspacePath: string,
+        rpcServerExplorer: RpcServerExplorer,
+        disposables: vscode.Disposable[]) {
+
+        this.rpcServerExplorer = rpcServerExplorer;
+
+        this.viewState = new ViewState();
+        this.viewState.path = workspacePath;
+        this.updateViewState();
+        
+        this.panel = vscode.window.createWebviewPanel(
+            'createInstancePanel',
+            'Create instance',
+            vscode.ViewColumn.Active,
+            { enableScripts: true });
+
+        this.panel.iconPath = vscode.Uri.file(path.join(extensionPath, 'resources', 'neo.svg'));
+
+        this.panel.onDidDispose(this.onClose, this, disposables);
+        this.panel.webview.onDidReceiveMessage(this.onMessage, this, disposables);
+
+        const htmlFileContents = fs.readFileSync(
+            path.join(extensionPath, 'src', 'panels', 'create.html'), { encoding: 'utf8' });
+        const javascriptHref : string = this.panel.webview.asWebviewUri(
+            vscode.Uri.file(path.join(extensionPath, 'out', 'panels', 'createBundle.js'))) + '';
+        const cssHref : string = this.panel.webview.asWebviewUri(
+            vscode.Uri.file(path.join(extensionPath, 'out', 'panels', 'create.css'))) + '';
+        this.panel.webview.html = htmlFileContents
+            .replace(JavascriptHrefPlaceholder, javascriptHref)
+            .replace(CssHrefPlaceholder, cssHref);
+    }
+
+    public isDisposed() {
+        return this.disposed;
+    }
+
+    public reveal() {
+        this.panel.reveal();
+    }
+
+    private onClose() {
+        this.dispose();
+    }
+
+    private async onMessage(message: any) {
+        if (message.e === createEvents.Init) {
+            this.panel.webview.postMessage({ viewState: this.viewState });
+        } else if (message.e === createEvents.Update) {
+            this.viewState = message.c;
+            this.updateViewState();
+            this.viewState.showError = false;
+            this.panel.webview.postMessage({ viewState: this.viewState });
+        } else if (message.e === createEvents.Create) {
+            this.doCreate();
+        } else if (message.e === createEvents.Close) {
+            this.dispose();
+        }
+    }
+
+    private updateViewState() {
+        this.viewState.combinedPath = path.join(this.viewState.path, this.viewState.filename);
+        this.viewState.configFileExists = fs.existsSync(this.viewState.combinedPath);
+        if (!this.viewState.configFileExists) {
+            this.viewState.allowOverwrite = false;
+        }
+    }
+
+    private doCreate() {
+        let command = shellEscape.default([
```

Comment:
Probably not important for create command, but let me know if you want any of these commands to provide machine readable output

Reply:
yeah, for now I take your stdout and render it to the user, but we might get to some commands where that is not very useful...